package.json

```diff
@@ -64,7 +64,8 @@
     "vue-i18n": "^8.0.0",
     "vue-js-toggle-button": "^1.2.2",
     "vue-notification": "^1.3.6",
-    "vue-select": "^3.0.2"
+    "vue-select": "^3.0.2",
+    "vuejs-datepicker": "^1.5.4"
```

Comment:
Please don't add ``vuejs-datepicker``, we already use ``@hokify/vuejs-datepicker``.
You can use ``<form-date>`` component which use it